lib/rules/jsx-indent-props.js

```diff
@@ -153,7 +153,7 @@ module.exports = {
 
       const indent = regExp.exec(src);
       const useOperator = /^([ ]|[\t])*[:]/.test(src) || /^([ ]|[\t])*[?]/.test(src);
-      const useBracket = /^([ ]|[\t])*[<]/.test(src);
```

Comment:
Can we add some test cases that use tabs, since that's what's being removed here?

Reply:
Yes, I've added them.